### ISSUE TITLE
Improvement at tuples test. 

### ISSUE THREAD

**Issue by DavidNV** [gregmalcolm/python_koans#195]
Hey @gregmalcolm. I hope this post finds you well. 
Thanks for maintaining this awesome project. 

While solving the Koans when I saw something at the `about_tuples.py` file: 

### Now
```python
def test_tuples_of_one_look_peculiar(self):
    self.assertEqual(__, (1).__class__)
    self.assertEqual(__, (1,).__class__)
    self.assertEqual(__, ("I'm a tuple",))
    self.assertEqual(__, ("Not a tuple"))
```
I'm not sure if that's the expected behaviour but I think it will be an improvement to check the class at the last two asserts.  Doing that will help those who do the koans to understand that the trailing comma means a `tuple` even if the value is not there and when it's just an element, it behaves as the value that is contained inside the `tuple`

### Suggestion

```python
def test_tuples_of_one_look_peculiar(self):
    self.assertEqual(__, (1).__class__)
    self.assertEqual(__, (1,).__class__)
    self.assertEqual(__, ("I'm a tuple",).__class__)
    self.assertEqual(__, ("Not a tuple").__class__)
```

I'll open a PR with it just in case you think it's a good idea too.

**Comment by hugovk** [comment]
Can this be closed now #196 is merged?

**Comment by gregmalcolm** [comment]
@hugovk i would say yes. Thanks everyone!
